layers/core_checks/device_memory_validation.cpp

```diff
@@ -851,6 +879,18 @@ bool CoreChecks::PreCallValidateUnmapMemory(VkDevice device, VkDeviceMemory mem)
     return skip;
 }
 
+bool CoreChecks::PreCallValidateUnmapMemory2KHR(VkDevice device, const VkMemoryUnmapInfoKHR* pMemoryUnmapInfo) const {
+    bool skip = false;
+    auto mem_info = Get<DEVICE_MEMORY_STATE>(pMemoryUnmapInfo->memory);
+    if (mem_info && !mem_info->mapped_range.size) {
+        // Valid Usage: memory must currently be mapped
+        skip |= LogError(pMemoryUnmapInfo->memory,
+                         "VUID-VkMemoryUnmapInfoKHR-memory-07964", "Unmapping Memory without memory being mapped: %s.",
```

Comment:
FFTI but given the simplicity of the vuid/fix for a vulkan dev, I think a concise message is best.

```suggestion
                         "VUID-VkMemoryUnmapInfoKHR-memory-07964", "Invalid memory unmapping: %s.",
```